meson.build

```diff
@@ -0,0 +1,67 @@
+project(
+  'flatpak-builder',
+  'c',
+  license: 'LGPL-2.1-or-later',
+  meson_version: '>= 0.56.2',
+  version: '1.3.1',
```

Comment:
I might recommend using a `files('VERSION')` paradigm here. Makes it easy to share a version between autotools and Meson I would think

Reply:
Given that the Autotools build is going to be removed in a not too distant future, I'm happy not to intertwine both in any meaningful way.